components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -272,6 +276,19 @@ def _configure(self, parser):
             "fileset",
             type=ProxyStringType("Fileset"))
 
+        logfile = parser.add(sub, self.logfile)
+        logfile.add_argument("fileset", type=ProxyStringType("Fileset"))
+        logfile.add_argument(
+            "filename",  nargs="?", default="-",
+            help="Local filename to be saved to. '-' for stdout")
```

Comment:
Yes, this was probably cut and paste from another command originally. As with the line length we have a lot of inconsistencies. I'm not sure if they should necessarily fixed in this PR - assuming one style is preferred. I do have a card to improve some of the CLI help text when some wider chnages could be made?